src/ValueFormatters/QuantityFormatter.php

```diff
@@ -38,6 +38,12 @@ class QuantityFormatter extends ValueFormatterBase {
 	const OPT_APPLY_ROUNDING = 'applyRounding';
 
 	/**
+	 * Option key controlling whether the quantity's unit of measurement should be included
+	 * in the output.
+	 */
+	const OPT_APPLY_UNIT = 'applyUnit';
```

Comment:
Hm, ok. Note that one reason for my confusion is the complexity of `QuantityFormatter::format`. This method should be simple.